### PR TITLE
DR4Hub add ns to samples 2.7 stolostron/backlog#26764

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26764

Please check the very first paragraph as the requested change only applies to 2.6. Let me know if the intro paragraph as it currently stands is still accurate for 2.7.

### CODE REVIEW

File: backup_restore/manage_backup_restore.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [#manage-backup-and-restore]
 = Managing the backup and restore operator
 
-The cluster backup and restore operator is not installed automatically. Enable the backup component by setting the `cluster-backup` parameter to `true`, in the `MultiClusterHub` resource. The cluster backup operator is installed in the `open-cluster-management-backup` namespace, where {product-title-short} is installed. When you install the cluster backup operator, the OADP Operator is also automatically installed.
+The cluster backup and restore operator is not installed automatically. Enable the backup component by setting the `cluster-backup` parameter to `true`, in the `MultiClusterHub` resource. When enabled, the cluster backup and restore operator is not installed in the `open-cluster-management-backup` namespace. When you install the cluster backup operator, the OADP Operator is also automatically installed.
```
Comment (birsanv) [inline backup_restore/manage_backup_restore.adoc]: - is not installed << this is wrong .. it should be is installed ( without *not)
- Replace
 When you install the cluster backup operator, the OADP Operator is also automatically installed.
with 
When you install the cluster backup operator, the OADP Operator is also automatically installed in the same namespace with the cluster backup and restore operator.

Everything else is good

**Review by birsanv — CHANGES_REQUESTED** [review]

File: backup_restore/manage_backup_restore.adoc (hunk 1 above)
Comment (oafischer) [reply backup_restore/manage_backup_restore.adoc]: Fixed, let me know if it looks good now, thanks!

**Review by birsanv — APPROVED** [review]
lgtm

File: backup_restore/manage_backup_restore.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [#manage-backup-and-restore]
 = Managing the backup and restore operator
 
-The cluster backup and restore operator is not installed automatically. Enable the backup component by setting the `cluster-backup` parameter to `true`, in the `MultiClusterHub` resource. The cluster backup operator is installed in the `open-cluster-management-backup` namespace, where {product-title-short} is installed. When you install the cluster backup operator, the OADP Operator is also automatically installed.
+The cluster backup and restore operator is not installed automatically. Enable the backup component by setting the `cluster-backup` parameter to `true`, in the `MultiClusterHub` resource. When enabled, the cluster backup and restore operator is installed in the `open-cluster-management-backup` namespace. When you install the cluster backup operator, the OADP Operator is also automatically installed in the same namespace as the cluster backup and restore operator.
```
Comment (cadawson) [inline backup_restore/manage_backup_restore.adoc:4]: Should we spell out OADP?

**Review by cadawson — APPROVED** [review]
Added a question, but it looks ok. Thanks!

**Comment by birsanv** [comment]
@cadawson `OADP Operator` is the name of the operator as showing in the Operator hub

**Pull request merged**